test/zdtm/static/memfd-secret00.desc

```diff
@@ -0,0 +1 @@
+{'feature': 'memfd_secret', 'flags': 'noauto'}
```

Comment:
I think this belongs to the commit that adds the test.

Reply:
I agree, thanks!